### ISSUE TITLE
Terraform Cloud Support In Azure-Pipelines Terraform Extension

### ISSUE THREAD

**Issue by sheldonhull** [hashicorp/azure-pipelines-extension-terraform#7]
https://github.com/microsoft/azure-pipelines-extensions/issues/754

Referenced the issue I opened. There seems to be no knowledge on the contents from this blog post and they recommended I reach out to Terraform directly. This seems the right place. 

I've been eagerly waiting for these improvements to simplify remote backend with terraform + automation on deploying and creating workspaces/variables from my azure pipelines using the terraform extension. However, I'm not finding anyone knowing anything about the content of the blog post mentioning these soon to be released improvements. Can someone help me understand what the timeline/progress on these improvements is?

Original Issue content from [Issue 754 in azure-pipelines-extensions](https://github.com/microsoft/azure-pipelines-extensions/issues/754
)

> Terraform Blog announced upcoming improvements with the azure pipelines extensions for Terraform that would support the following types of commands:
> 
> Terraform Cloud Backend
> Create new tfe_workspace, variables, and more.
> Initiating remote run, plan, apply.
> I've been searching issues, announcements, community discussion, and even commits, but am having difficulty in finding anything on eta, progress, or issues to watch on this.
> 
> I need ways to improve the terraform cloud workflow and am hoping azure pipelines can offer this, so I can automate workspace management.
> 
> Can someone provide some linked issues on implementation of Terraform Cloud support with the terraform extension so I can be more aware of the progress of this new feature?

**Comment by sukoneck** [comment]
I had read the same [article](https://www.hashicorp.com/blog/announcing-azure-devops-support-for-terraform-cloud-and-enterprise/) as @sheldonhull and was disappointed to learn that ADO support for TC backend [is not](https://github.com/microsoft/azure-pipelines-extensions/issues/754#issuecomment-567371162) even on their public backlog. My team built out the tasks to run as bash commands, but it's clunky. I'm hoping this reconnects the Hashi/ADO teams and we see some movement.

**Comment by sheldonhull** [comment]
Agreed. Blog post was very confusing. TBH it is probably better that we all just build some azure devops templates in yaml to handle this outselves.

**Comment by IvanJosipovic** [comment]
+1
